molecule_ec2/cookiecutter/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/create.yml

```diff
@@ -73,53 +73,49 @@
         filters:
           name: "{{ item.image_name }}"
       loop: "{{ molecule_yml.platforms }}"
-      when: item.image is not defined
+      when: item.image_id is not defined
       register: ami_info
 
     - name: Create molecule instance(s)
-      ec2:
+      ec2_instance:
```

Comment:
is there any special reason we are replacing the ec2 module with ec2_instance?

Reply:
`The ec2 module documentation should probably be updated to suggest using the ec2_instance module.`

From this issue: https://github.com/ansible/ansible/issues/64610#issuecomment-551924294

Reply:
makes sense, thanks for sharing